shared/templates/create_audit_rules_privileged_commands.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/python2
+
+#
+# create_audit_rules_privileged_commands.py
+#        generate template-based checks for privileged commands rules
+
+
+from template_common import FilesGenerator, UnknownTargetError
+
+import re
+import ntpath
+def path_leaf(path):
+    head, tail = ntpath.split(path)
+    return tail or ntpath.basename(head)
+
+
+class AuditRulesPrivilegedCommandsGenerator(FilesGenerator):
+    def generate(self, target, args):
+        path = args[0]
+        name = re.sub('[-\./]', '_', path_leaf(path))
+        if target == "oval":
+            self.file_from_template(
+                "./template_OVAL_audit_rules_privileged_commands",
+                {
+                    "@NAME@":	name,
+                    "@PATH@":	re.sub("/", "\/", path)
```

Comment:
We use `@VARIABLE@` if the generic build system replaces it and `%VARIABLE%` if the template replaces it. Please follow that convention.